test/JDBC/expected/babel_function_string-vu-verify.out

```diff
@@ -385,6 +385,78 @@ nvarchar
 ~~END~~
 
 
+SELECT QUOTENAME('Hello ) there', ')')
+GO
+~~START~~
+nvarchar
+(Hello )) there)
+~~END~~
+
+
+SELECT QUOTENAME('Hello )" there', ')')
+GO
+~~START~~
+nvarchar
+(Hello ))" there)
+~~END~~
+
+
+SELECT QUOTENAME('Hello < there', '<')
+GO
+~~START~~
+nvarchar
+<Hello < there>
+~~END~~
+
+
+SELECT QUOTENAME('Hello > there', '>')
+GO
+~~START~~
+nvarchar
+<Hello >> there>
+~~END~~
+
+
+SELECT QUOTENAME('Hello } there', '}')
+GO
+~~START~~
+nvarchar
+{Hello }} there}
+~~END~~
+
+
+SELECT QUOTENAME('Hello ` there', '`')
+GO
+~~START~~
+nvarchar
+`Hello `` there`
+~~END~~
+
+
+SELECT QUOTENAME('[KEY]', '"')
+GO
+~~START~~
+nvarchar
+"[KEY]"
+~~END~~
+
+
+SELECT QUOTENAME('"KEY"', '[')
+GO
+~~START~~
+nvarchar
+["KEY"]
+~~END~~
+
+
+SELECT QUOTENAME('"KEY"', '')
```

Comment:
can you please add some test cases for multibyte characters? with various combination with length